Database/Table Scripts/CreatePetServiceTableScript.sql

```diff
@@ -0,0 +1,12 @@
+IF (EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND TABLE_NAME = 'PetServices'))
```

Comment:
Table that will store the various pet services like pet sitting, walks, check-ins..etc.

This table has the price and the rate employees will be paid.

The parameters for DECIMAL is 

- 1st parameter indicates the max length of entire decimal value
- 2nd parameter indicates how many trailing numbers behind decimal.

For example, for Decimal(5,2), this means the decimal value can be max 5 digits with 2 digits trailing behind the decimal point. This is good for percentages because it encompasses the max we will need to represent 100%

- 100.00 <--- max 5 digits with 2 trailing digits after decimal point